mmv1/templates/terraform/examples/node_template_disks.tf.tmpl

```diff
@@ -0,0 +1,16 @@
+data "google_compute_node_types" "central1a" {
+  zone     = "us-central1-a"
+}
+
+resource "google_compute_node_template" "{{$.PrimaryResourceId}}" {
+  name      = "{{index $.Vars "template_name"}}"
+  region    = "us-central1"
+  node_type = "n2-node-80-640"
+
+  disks {
```

Comment:
Can you add one more `disks` block to test if multiple disks work as expected?

Reply:
The API itself is misleading because it does seem to support multiple sets of disk pools, however in actual application (at least for the past two years I've been working with sole-tenant nodes) only one type of disk is available for selection (`local-ssd`) and you have only three choices: None, 16/6TB, 24/9TB.

![image](https://github.com/user-attachments/assets/bb00321e-5054-415c-bb16-e83926931fef)

Even the api field descriptions are somewhat inaccurate. Take `diskSizeGb` for example: "integerSpecifies the size of the disk in base-2 GB." However this must only be `375` which is the partition size of local-ssd disks.

The API will only use the last `--disk` argument passed in:

```sh
gcloud compute sole-tenancy node-templates create local-ssd-test \
  --node-type=n2-node-80-640 \
  --region=us-central1 \
  --node-affinity-labels="foo=bar" \
  --disk type=local-ssd,count=16,size=375 \
  --disk type=local-ssd,count=24,size=375
```

```sh
> gcloud compute sole-tenancy node-templates describe local-ssd-test
cpuOvercommitType: NONE
creationTimestamp: '2024-11-02T08:02:56.823-07:00'
disks:
- diskCount: 24
  diskSizeGb: 375
  diskType: local-ssd
id: 'xxxxx'
kind: compute#nodeTemplate
name: local-ssd-test
nodeAffinityLabels:
  foo: bar
nodeType: n2-node-80-640
region: https://www.googleapis.com/compute/v1/projects/xxxxx/regions/us-central1
selfLink: https://www.googleapis.com/compute/v1/projects/xxxxx/regions/us-central1/nodeTemplates/local-ssd-test
serverBinding:
  type: RESTART_NODE_ON_ANY_SERVER
status: READY
```
![image](https://github.com/user-attachments/assets/7c878a45-7dbc-4d72-900a-d66ba934e1e9)